tests/services/schemas/test_resource_type.py

```diff
@@ -20,34 +20,45 @@
 #       when testing for a valid minimal metadata as a whole
 
 
-def test_valid_no_subtype(minimal_metadata, vocabulary_clear):
+def test_valid_type_and_subtype(minimal_metadata):
+    # default minimal_metadata's resource type is image-photo
+    expected_metadata = deepcopy(minimal_metadata)
+    expected_metadata["creators"][0]["person_or_org"]["name"] = "Brown, Troy"
+
+    assert expected_metadata == MetadataSchema().load(minimal_metadata)
+
+
+def test_valid_no_subtype(minimal_metadata):
+    # whether id represents a resource type with or without subtype should
+    # not matter
     expected_metadata = deepcopy(minimal_metadata)
     expected_metadata["creators"][0]["person_or_org"]["name"] = "Brown, Troy"
     expected_metadata["resource_type"] = minimal_metadata["resource_type"] = {
-        "type": "poster"
+        "id": "dataset"
     }
 
     assert expected_metadata == MetadataSchema().load(minimal_metadata)
 
 
-def test_invalid_no_resource_type(minimal_metadata, vocabulary_clear):
-    minimal_metadata["resource_type"] = {}
-    assert_raises_messages(
-        lambda: MetadataSchema().load(minimal_metadata),
-        {"resource_type": ["Missing data for required field."]}
-    )
+# TODO: Reactivate when we deal with frontend
```

Comment:
The frontend has been merged, should be fairly straightforward... But otherwise, I would open a ticket... I fear skipped and commented tests are never being re-activated.